src/timeline/InputBar.cpp

```diff
@@ -846,6 +903,8 @@ InputBar::command(const QString &command, QString args)
                                 cache::getMembers(this->room->roomId().toStdString(), 0, -1));
     } else if (command == QLatin1String("converttoroom")) {
         utils::removeDirectFromRoom(this->room->roomId());
+    } else {
+        message("/" + command + " " + args);
```

Comment:
this actually causes linebreas to get converted into a space, when a message starts with a / :D